viewer/__init__.py

```diff
@@ -244,9 +244,9 @@ def thingview(path, suffix=None):
 # FIXME make this less sketcy
 def _get_view_data_accept_header(request, suffix):
     mimetype, _ = negotiator.negotiate(request, suffix)
-    if mimetype in ('application/json'):
```

Comment:
This was the actual culprit. Testing for None in a tuple is OK, looking for None in a string is not.